app/assets/stylesheets/alchemy/admin/tables.scss

```diff
@@ -195,13 +195,17 @@ td,
   }
 }
 
-th.count,
-td.count {
+th.taggings_count,
```

Comment:
Why has this classed be renamed? It might be used by extensions and should not be altered if not necessary

Reply:
It is now count again and I added the possibility to add every CSS class